tests/test_validators.py

```diff
@@ -37,3 +38,39 @@ def test_validate_merged_with_fails(merge_candidates):
 
     with pytest.raises(ValidationError):
         validate_merged_with(primary.id)
+
+
+@pytest.mark.django_db
+def test_validate_merged_fails_when_name_merges_with_itself():
```

Comment:
This shoud be named `test_validate_merged_with_fails_when_name_merges_with_itself()`